pup-0006.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Commit Bit Assignment
+author: Brian Bouterse
+created: 01-06-2018
+status: Approved
+---
+
+## Summary
+
+For pulpcore and its related tools, there is no written process to describe giving the commit bit to
+a contributor for all or a portion of the codebase. The existing committers want to openly document
+the process.
+
+
+## Motivation
+
+Historically the commit bit was given on day 0 when hired on the core team at Red Hat.  In Oct 2017
+the existing committers decided to stop that practice and instead document an open process.
+Engineers hired on the Pulp team since Oct '17 have not received commit bit.
+
+We have not yet documented an open process of which to give a proven contributor commit access to
+all or a portion of the codebase.
+
+
+## Scope
+
+This formally applies to the following core repositories and core-team maintained plugins:
+
+Core Repos:
+* pulp/pulp
+* pulp/devel
+* pulp/pulp-ci
+
+Plugin Repos:
+* pulp/pulp_file
+* pulp/pulp_python
+* pulp/pulp_rpm
+* pulp/crane
+* pulp/pulp_docker
+* pulp/pulp_puppet
+* pulp/pulp_ostree
+
+This policy is intended as the default for all software in the Pulp ecosystem unless explicitly
+stated otherwise.
+
+
+## Criteria
+
+Overall the candidate must have demonstrated commitment and care towards the needs of all Pulp
+users and not only their own interests. Also they must have the experience to be trusted with major
+aspects of Pulp functionality in that area.
+
+These requirements are somewhat vague by design and leave the decision to the judgement of the
```

Comment:
i do think that we need to write down some specific criteria.
from email thead ' i suggest we list at least some kind of criteria like X time involved in project development, has demonstrated to provide good form and useful code reviews, has submitted code that was accepted into main parts of the project.'